### PR TITLE
Update Kconfig: Configuring environment variables to run the edge-orc…

### PR DESCRIPTION
**Pull request by tdrozdovsky**
…hestration

Signed-off-by: Taras Drozdovskyi <t.drozdovsky@samsung.com>

# Description

For the convenience of launching the edge-orchestration container, the ability to configure the environment parameters using the configuration file has been added.

These changes do not affect the build system of the edge-orchestration, only the launch parameters. 

## Type of change

Please delete options that are not relevant.

- [x] Code cleanup/refactoring
- [x] CI system update

# How Has This Been Tested?

```
make menuconfig
```
image

```
make run V=1

--------------------------------------
  Run Docker container 
--------------------------------------
docker run -it -d \
                --privileged \
                --network="host" \
                --name edge-orchestration \
                -e MNEDC=server -e SECURE=true -e WEBUI=true -e LOGLEVEL=Info \
                -v /var/edge-orchestration/:/var/edge-orchestration/:rw \
                -v /var/run/docker.sock:/var/run/docker.sock:rw \
                -v /proc/:/process/:ro \
                "lfedge/edge-home-orchestration-go":"latest"

```

**Test Configuration**:
* OS type & version: Ubuntu 20.04)
* Hardware: x86-64
* Toolchain: Docker v20.10 and Go v1.16
* Edge Orchestration Release: v1.1.4

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: configs/defconfigs/x86_64n [hunk 1]
```diff
@@ -10,5 +10,22 @@ CONFIG_X86_64=y
...
 # CONFIG_ANDROID is not set
+
+#
+# Configuring environment variables to run the Edge-Orchestration (make run)
+#
 # CONFIG_MNEDC is not set
-# CONFIG_SECURE_MODE is not set
+# CONFIG_MNEDC_SERVER is not set
+# CONFIG_MNEDC_CLIENT is not set
+CONFIG_SECURE_MODE=y
```
Comment (t25kim) [inline configs/defconfigs/x86_64n:20]: Is it right to set `CONFIG_SECURE_MODE` to `y`?

**Review by t25kim — COMMENTED** [review]
Thanks for the contribution! PTAL!

File: Kconfig [hunk 2]
```diff
@@ -90,6 +92,73 @@ config SECURE_MODE
 	---help---
 		"Secure mode is enable"
 
+config WEB_UI
+	bool "Web UI is enable"
+	default y
```
[the remark this answers is not in the document]
Comment (tdrozdovsky) [reply Kconfig]: Thank you! Done!

**Review by t25kim — APPROVED** [review]
LGTM Thanks!

**Review by MoonkiHong — APPROVED** [review]
Looks good to me. 😁

**Review by suresh-lc — APPROVED** [review]
Config values for various modes is in synch with current feature set.

**Pull request merged**